components/Error.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import Layout from './Layout'
+import NavLink from './NavLink'
+import { Text } from './theme/Text'
+export enum StatusCode {
+  NOT_FOUND = '404',
```

Comment:
Why are the `status code numbers` strings? I think it would make more sense to use numbers as numbers, rather than as strings.

Additionally, this removes the need to have a `toStatusCode` function that you have in your PR#485